CMakeLists.txt

```diff
@@ -640,7 +640,7 @@ function(get_link_libraries OUT TARGET)
     set(RESULT "")
     get_target_property(LIBRARIES ${TARGET} INTERFACE_LINK_LIBRARIES)
     foreach(LIB ${LIBRARIES})
-	if("${LIB}" MATCHES ".*NOTFOUND.*")
+	if(NOT LIB)
```

Comment:
I reckon NOTFOUND is different to unset. Not sure though. Do you have a link to the docs at hand?

Reply:
Yes, they are different, but both false. See https://cmake.org/cmake/help/v3.15/command/if.html, section "if(<constant>)".